ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -495,4 +511,37 @@ private void displayWelcomeMessage() {
 
     writer.flush();
   }
+
+  static void maybeCreateProcessingLogStream(
+      final ProcessingLogConfig config,
+      final KsqlConfig ksqlConfig,
+      final KsqlEngine ksqlEngine,
+      final CommandQueue commandQueue) {
+    if (!config.getString(ProcessingLogConfig.STREAM_AUTO_CREATE).equals(
+        ProcessingLogConfig.AUTO_CREATE_ON)) {
+      return;
+    }
+    final String name = config.getString(ProcessingLogConfig.STREAM_NAME);
+    final String topicName = config.getString(ProcessingLogConfig.TOPIC_NAME);
+    final String statementText = ProcessingLogServerUtils.processingLogStreamCreateStatement(
+        name,
+        topicName);
+    if (!commandQueue.isEmpty()) {
+      return;
+    }
+    final List<PreparedStatement<?>> statement;
+    try {
+      statement = ksqlEngine.parseStatements(statementText);
+      ksqlEngine.tryExecute(statement, ksqlConfig, Collections.emptyMap());
+    } catch (final KsqlException e) {
+      log.warn("Failed to create processing log stream", e);
+      return;
+    }
+    commandQueue.enqueueCommand(
```

Comment:
This is queuing a command to the command topic on each server start, for each server in the cluster, right?

Is this intentional?  Why not just execute this against the engine directly as we do for the command topic?